minerl/Malmo/Minecraft/gradlew.bat

```diff
@@ -8,14 +8,14 @@
 @rem Set local scope for the variables with windows NT shell
 if "%OS%"=="Windows_NT" setlocal
 
-@rem Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-set DEFAULT_JVM_OPTS=
```

Comment:
Same comment here. As in, it looks like the code below should not be affected by DEFAULT_JVM_OPTS at all. Was it affected?